sample/HttpTrigger-Powershell-Modules/modules/show-calendar.psm1

```diff
@@ -0,0 +1,147 @@
+<# 

```

Comment:
nit: can you normalize the casing of your file names? This one is lower, it's peer is upper.